### PR TITLE
clarity readme

### PR DESCRIPTION
**Pull request by grosser**
@adam-hart 
/cc @zendesk/compute

### CODE REVIEW

File: plugins/kubernetes/README.md [hunk 1]
```diff
@@ -132,11 +132,11 @@ Via [Template filler](/plugins/kubernetes/app/models/kubernetes/template_filler.
 
 ### Migrations / Prerequisite
 
-Add a role with only a `Pod`, `metadata.annotations.samson/prerequisite: 'true'`, and command to run a migrations.
-It will be executed before the rest is deployed.
+Add a role with a `Pod` or `Job`, `metadata.annotations.samson/prerequisite: 'true'`, and `command` to run a migrations.
+It will be executed before any other role is deployed.
```
Comment (adammw) [inline plugins/kubernetes/README.md]: still not clear you need to add this to the job and not the pod template

**Comment by grosser** [comment]
added "root" ... better ?

File: plugins/kubernetes/README.md (hunk 1 above)
Comment (hgokavarapuz) [reply plugins/kubernetes/README.md]: same here...That description is still not clear.

File: plugins/kubernetes/README.md [hunk 2]
```diff
@@ -132,11 +132,11 @@ Via [Template filler](/plugins/kubernetes/app/models/kubernetes/template_filler.
 
 ### Migrations / Prerequisite
 
-Add a role with only a `Pod`, `metadata.annotations.samson/prerequisite: 'true'`, and command to run a migrations.
-It will be executed before the rest is deployed.
+Add a role with a `Pod` or `Job`, root `metadata.annotations.samson/prerequisite: 'true'`, and command to run a migrations.
+It will be executed before any other role is deployed.
```
Comment (adammw) [inline plugins/kubernetes/README.md]: Suggestion:

> Migrations/prerequisites should be added as a separate role with the annotation  `samson/prerequisite: 'true'` set on your `Job`/`Deployment`/`Pod` (annotation should be added to the 'root' object, not the template). This role will be deployed/executed before any other role is deployed.

Comment (grosser) [reply plugins/kubernetes/README.md]: updated

**Review by adammw — APPROVED** [review]

**Pull request merged**